extension/src/openvic-extension/classes/GFXButtonStateTexture.cpp

```diff
@@ -60,17 +59,14 @@ Ref<GFXButtonStateTexture> GFXButtonStateTexture::make_gfx_button_state_texture(
 	ERR_FAIL_NULL_V(button_state_texture, nullptr);
 	button_state_texture->set_button_state(button_state);
 	if (source_image.is_valid()) {
-		ERR_FAIL_COND_V(
-			button_state_texture->generate_state_image(source_image, region, cornered_tile_border_size) != OK, nullptr
-		);
+		ERR_FAIL_COND_V(button_state_texture->generate_state_image(source_image, region, cornered_tile_border_size) != OK, nullptr);
```

Comment:
This line has been recombined to now have a column width over 128, while the function call on line 28 wasn't recombined fully (presumably as that would make it over 128 columns wide). Is there any way to make this consistent?

Reply:
No because clang-format has no semantic information, its decisions are completely based on independent line decisions.